### PR TITLE
Add connector_jira_tempo_validate_ts_ee

### PR DESCRIPTION
**Pull request by simahawk**
C2C-649
depends on 
https://github.com/camptocamp/connector-jira/pull/41
#10 (so the base branch is the one of that PR)

### CODE REVIEW

File: connector_jira_tempo_validate_ts_ee/models/jira_backend/common.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+from odoo import models
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class JiraBackend(models.Model):
+    _inherit = 'jira.backend'
+
+    def _validate_ts(self, date_from, date_to, state, user_ids):
```
Comment (i-vyshnevska) [inline connector_jira_tempo_validate_ts_ee/models/jira_backend/common.py]: should it run for employees without users?

Comment (simahawk) [reply connector_jira_tempo_validate_ts_ee/models/jira_backend/common.py]: a.a.l. are based on users, there's no way AFAIS to submit them w/out a user.
In any case, here, in the context of Jira sync, they ARE tied to users only.
Am I missing anything?

Comment (i-vyshnevska) [reply connector_jira_tempo_validate_ts_ee/models/jira_backend/common.py]: here you probably can't submit without user, but by default it's possible to have employee without users and grid works with employees not users, 
I don't have user in odoo :)

Comment (simahawk) [reply connector_jira_tempo_validate_ts_ee/models/jira_backend/common.py]: of course it works w/ employee but they are linked via the user of the employee.
You can check in `odoo/src/addons/hr_timesheet/models/hr_timesheet.py`.
And it seems pretty logic: a TS makes sense if there's an employee - intended as a human being / person / user - that has to track his/her own time.

**Pull request closed without merging**